src/sagemaker/tensorflow/estimator.py

```diff
@@ -201,6 +201,7 @@ class TensorFlow(Framework):
     __framework_name__ = "tensorflow"
 
     LATEST_VERSION = defaults.LATEST_VERSION
+    LATEST_SERVING_VERSION = defaults.LATEST_SERVING_VERSION
```

Comment:
probably don't need this in the estimator - you can import it from `sagemaker.tensorflow.defaults`